ImageProcessing/README.md

```diff
@@ -0,0 +1,747 @@
+
+1. Click **"Create Change Set"** in the Transforms section
+
+1. Click **"Execute"**
+
+1. Wait for the `wildrydes-step-module-resources` stack to reach a status of `CREATE_COMPLETE`.
+
+1. With the `wildrydes-step-module-resources` stack selected, click on the **Outputs** tab. These resources will be referenced in the later steps. 
+
+</p></details>
+
+
+
+### 3. Create an initial AWS Step Functions state machine
+
+After the riders upload their photo, the first thing we need do in our processing pipeline is to run a face detection algorithm on it to verify that it has a recognizable face in the photo (zero or multiple faces in the photo doesn't help unicorns recognize the rider) and the face is not wearing sunglasses (makes it harder to identify the rider). If these validations fail, notify the user and end the workflow.
+
+The AWS Lambda function that implements this check by leveraging the **Amazon Rekognition** deep-learning based image analysis API is already deployed by AWS CloudFormation in the previous step. Look in the **Outputs** section for `FaceDetectionFunctionArn` for the ARN of the Lambda function. 
+
+The AWS Lambda function to be called when the validations fail is deployed by the AWS CloudFormation called `NotificationPlaceholderFunction`. The intent behind this step is to notify the user the photo validation failed and the error reason, so they can try upload a different photo. To keep this workshop module short, the function just prepares the message instead of actually sending the message (feel free to modify the code to send email notifications through SES)
+
+Now you can create an AWS Step Functions State Machine with the initial face detection step: 
+
+![initial state machine diagram](./images/1st-state-machine-graph.png)
+
+<details>
+<summary><strong>Step-by-step instructions (expand for details)</strong></summary><p>
+
+1. AWS Step Functions state machine flows are defined by a JSON document. In your favorite text editor, create a file called `rider-photo-state-machine.json`
+
+1. copy and paste the following into your JSON file:
+
+	```JSON
+	{
+	  "Comment": "Rider photo processing workflow",
+	  "StartAt": "FaceDetection",
+	  "States": {
+	    "FaceDetection": {
+	      "Type": "Task",
+	      "Resource": "REPLACE_WITH_FaceDetectionFunctionArn",
+	      "ResultPath": "$.detectedFaceDetails",
+	      "End": true,
+	      "Catch": [
+	        {
+	          "ErrorEquals": [
+	            "PhotoDoesNotMeetRequirementError"
+	          ],
+	          "ResultPath": "$.errorInfo",
+	          "Next": "PhotoDoesNotMeetRequirement"
+	        }
+	      ]
+	    },
+	    "PhotoDoesNotMeetRequirement": {
+	      "Type": "Task",
+	      "Resource": "REPLACE_WITH_NotificationPlaceholderFunctionArn",
+	      "End": true
+	    }
+	  }
+	}
+
+	```
+	> The above JSON defines a state machine using the [Amazon States Language](https://states-language.net/spec.html). Take a moment to understand its structure. 
```

Comment:
it was intentional, but i agree that there's no reason to do so. made it normal text